File: tools-manage/invocationlab-admin/src/main/java/org/javamaster/invocationlab/admin/model/erd/ErdOnlineModel.java
package org.javamaster.invocationlab.admin.model.erd;

import lombok.Data;
import lombok.NoArgsConstructor;

/**
 * @author yudong
 * @date 2023/2/12
 */
@NoArgsConstructor
@Data
public class ErdOnlineModel {
    private ConfigJSONBean configJSON;
    private ProjectJSONBean projectJSON;
    private String projectName;
    private String type;
    private String id;
}

File: tools-manage/invocationlab-admin-21/src/main/kotlin/org/javamaster/invocationlab/admin/model/erd/ErdOnlineModel.kt
package org.javamaster.invocationlab.admin.model.erd

import org.javamaster.invocationlab.admin.annos.AllOpen

/**
 * @author yudong
 * @date 2023/2/12
 */


@AllOpen
class ErdOnlineModel {
    var configJSON: ConfigJSONBean? = null
    var projectJSON: ProjectJSONBean? = null
    var projectName: String? = null
    var type: String? = null
    var id: String? = null
}
